model/histogram/float_histogram_test.go

```diff
@@ -222,7 +222,11 @@ func TestFloatHistogramMul(t *testing.T) {
 	}
 
 	for _, c := range cases {
+		// capture the range variable to avoid issues with concurrency
+		c := c
```

Comment:
This is the default in Go since 1.22 - do we need it?
https://go.dev/blog/loopvar-preview